bundles/org.openhab.core.compat1x/src/main/java/org/openhab/model/sitemap/Chart.java

```diff
@@ -27,6 +27,7 @@
  * @see org.openhab.model.sitemap.SitemapPackage#getChart()
  * @model
  * @generated
+ * @author Kai Kreuzer - Initial contribution
```

Comment:
Hm, I think we can move these files to `src/gen/java` as we have done in the OH2 addons build.